package.json

```diff
@@ -42,7 +42,7 @@
     "lodash": "^4.17.21",
     "pdfjs-dist": "2.0.943",
     "regenerator-runtime": "^0.13.9",
-    "sbc-common-components": "3.0.3",
+    "sbc-common-components": "3.0.5",
```

Comment:
I verified the changes from 3.0.3 to 3.0.4, and from 3.0.4 to 3.0.5 -- no issues.